### PR TITLE
Update migrate-addon script

### PR DESCRIPTION
**Pull request by howardjohn**
Internal bug: 204365008

### CODE REVIEW

**Comment by zerobfd** [comment]
Can you do the PR against main instead?

**Comment by howardjohn** [comment]
> Can you do the PR against main instead?

the script is only in `master` for whatever reason

**Comment by zerobfd** [comment]
You're right, I just can't read.

File: scripts/migration/migrate-addon [hunk 1]
```diff
@@ -203,7 +241,90 @@ EOF
...
+                sdsConfig:
+                  initialFetchTimeout: 1s
+                  apiConfigSource:
+                    apiType: GRPC
+                    grpcServices:
+                    - envoyGrpc:
+                        clusterName: sds-grpc
+EOF
+  echo "Waiting for proxies to pick up the new root certificate..."; sleep 15
+  kube delete envoyfilter trigger-root-cert -n istio-system
```
Comment (zerobfd) [inline scripts/migration/migrate-addon:313]: I guess there's not an easy way to actually check?

**Review by zerobfd — APPROVED** [review]
LGTM.

File: scripts/migration/migrate-addon (hunk 1 above)
Comment (howardjohn) [reply scripts/migration/migrate-addon:313]: Unfortunately not really. We can go exec into every pod but... not ideal

**Pull request merged**